### PR TITLE
[PassKit] Added support for Xcode 10 up to beta 3.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/PassKit/PKEnums.cs [hunk 1]
```diff
@@ -198,4 +198,13 @@ public enum PKPaymentErrorCode : long {
...
 		ShippingAddressUnserviceable,
 	}
+
+	[iOS (12,0)]
+	[NoWatch]
+	[Native]
+	public enum PKAddPaymentPassStyle : ulong
+	{
+		Payment,
+		Access
```
Comment (dalexsoto) [inline src/PassKit/PKEnums.cs]: missing comma

File: src/PassKit/PKEnums.cs [hunk 2]
```diff
@@ -198,4 +198,13 @@ public enum PKPaymentErrorCode : long {
 		BillingContactInvalid,
 		ShippingAddressUnserviceable,
 	}
+
+	[iOS (12,0)]
+	[NoWatch]
+	[Native]
+	public enum PKAddPaymentPassStyle : ulong
+	{
```
Comment (dalexsoto) [inline src/PassKit/PKEnums.cs]: Style, this should go one line up

**Comment by mandel-macaque** [comment]
@dalexsoto wil mix those minor and will merge

File: src/passkit.cs [hunk 3]
```diff
@@ -620,6 +620,10 @@ interface PKAddPaymentPassRequestConfiguration : NSSecureCoding
 		[NoWatch] // Radar: https://trello.com/c/MvaHEZlc
 		[Export ("requiresFelicaSecureElement")]
 		bool RequiresFelicaSecureElement { get; set; }
+
+		[iOS (12, 0)]
```
Comment (spouliot) [inline src/passkit.cs]: missing `[NoWatch]` here - the `enum` type has the attribute

**Pull request merged**